eng-wasm/src/crypto_wasm.rs

```diff
@@ -0,0 +1,57 @@
+use super::*;
+
+use rand_wasm::Rand;
+use eng_pwasm_abi::types::U256;
+
+const SYMMETRIC_KEY_SIZE: usize = 32;
+const IV_SIZE: usize = 96 / 8;
+const PAYLOAD_SIZE: usize = 1024;
+
+pub type IV = [u8; IV_SIZE];
+pub type SymmetricKey = [u8; SYMMETRIC_KEY_SIZE];
+
+pub fn generate_key() -> SymmetricKey {
+    let key_int: U256 = Rand::gen();
+    let key = H256::from(key_int);
+    key.0
+}
+
+pub fn encrypt(message: &[u8], key: &SymmetricKey) -> Vec<u8> {
+    // TODO: Is this really needed? Dynamically sized buffers don't seem to work.
+    // TODO: Is it possible to estimate the encrypted payload size based on the plaintext message?
```

Comment:
You can estimate the encrypted payload size. In block cypher it is equal to the plaintext message with completion of the last part to a block size.

Reply:
Are you ok if I keep the static size for now and create a new PR to implement this? I other things to finish before I can revisit this and I don't want to keep the PR open for too long.

Reply:
Sure